common/compositor/gl/glprogram.h

```diff
@@ -21,6 +21,8 @@
 
 #include "shim.h"
 
+#define PREBUILT_SHADER_FILE_PATH "/vendor/etc/"
```

Comment:
Instead of hardcoding this here, can we pass this from the build system?
For example in Android.mk -DPREBUILT_SHADER_FILE_PATH="/vendor/etc/"
On linux -DPREBUILT_SHADER_FILE_PATH="${prefix}/etc"

Reply:
ok I will move it to Makefile.